django_declarative_apis/machinery/filtering.py

```diff
@@ -280,4 +311,5 @@ def apply_filters_to_object(inst, filter_def, expand_header=""):
         expand_children=expand_dict,
         klass=inst.__class__,
         filter_cache={},
+        model_cache={},
```

Comment:
decided to add a new cache instead of reusing `filter_cache` since they key/values types are completely different from filter cache.

Filter cache using is storing
- key: (expandables, class, pk)
- value: defaultdict(...) with filtered/formatted results

Model cache:
- key: (class, pk)
- value: model instances